packages/speech/src/helpers.js

```diff
@@ -80,29 +79,39 @@ module.exports = () => {
     // Format the audio content as input request for pipeline
     var recognizeStream = streamEvents(pumpify.obj());
 
-    recognizeStream.once('writing', function() {
-      requestStream.on('error', function(err) {
+    // Attach the events to the request stream, but only do so
+    // when the first write (of data) comes in.
+    //
+    // This also means that the sending of the initial request (with the
+    // config) is delayed until we get the first burst of data.
+    recognizeStream.once('writing', () => {
```

Comment:
`() => {` binds the current context with it, which would be this instance of a Speech constructor. We'll have to change the later line "this.setPipeline" to "recognizeStream.setPipeline"